### PR TITLE
Add CodeQL fixes

### PR DESCRIPTION
**Pull request by tomato42**
Add fixes to make CodeQL analysis clean.

### CODE REVIEW

File: src/ecdsa/test_eddsa.py [hunk 1]
```diff
@@ -500,6 +500,7 @@ def test_equal_public_points(self):
         key2 = PublicKey(generator_ed25519, b"\x01" * 32)
 
         self.assertEqual(key1, key2)
+        # verify that `__neq__` works as expected
```
Comment (jogme) [inline src/ecdsa/test_eddsa.py:503]: typo; same below
```suggestion
        # verify that `__ne__` works as expected
```

**Review by jogme — CHANGES_REQUESTED** [review]

File: src/ecdsa/test_eddsa.py (hunk 1 above)
Comment (tomato42) [reply src/ecdsa/test_eddsa.py]: fixed

**Review by jogme — APPROVED** [review]
looking good!

**Comment by tomato42** [comment]
@jogme Thanks!

File: src/ecdsa/test_eddsa.py (hunk 1 above)
Comment (sajjantimalsina12) [reply src/ecdsa/test_eddsa.py]: Working id 3833dd2f3e-3584f_
_update key <21c4d0a390a7fc27>
Give me new  code_______

**Pull request merged**